go/vt/vttablet/tabletserver/vstreamer/uvstreamer_flaky_test.go

```diff
@@ -298,18 +298,29 @@ func TestVStreamCopyCompleteFlow(t *testing.T) {
 }
 
 func validateReceivedEvents(t *testing.T) {
+	inCopyPhase := false
 	for i, ev := range allEvents {
 		ev.Timestamp = 0
-		if ev.Type == binlogdatapb.VEventType_FIELD {
+		switch ev.Type {
+		case binlogdatapb.VEventType_OTHER:
+			if strings.Contains(strings.ToLower(ev.Gtid), "copy start") {
```

Comment:
Similar to a former comment, can we come up with a more formal check other than "Contains"?

Reply:
Again, no. These are unit tests where we're building things ourselves. This has no relation to production usage in that sense.

Reply:
I.e. this is a generic `OTHER` event that we're using specifically in the unit test to indicate that we're starting the copy phase. I'll add a constant at least to make this part a bit more clear.